server/authorizer/authorizer.go

```diff
@@ -32,6 +32,8 @@ type Basic struct {
 	printHook             print.Hook
 	enablePrintStatements bool
 	interQueryCache       cache.InterQueryCache
+	regexBuiltinCache     cache.ValueCache
+	globBuiltinCache      cache.ValueCache
```

Comment:
Is there a way to set these, similar to  `func InterQueryCache(interQueryCache cache.InterQueryCache) func(*Basic)`?

Reply:
Which should then be called in `server.Server.initHandlerAuthz()`?

Reply:
Yes, will update.

Reply:
Done